tests/interfaces/records/handlers/records-write.spec.ts

```diff
@@ -570,6 +572,60 @@ describe('RecordsWriteHandler.handle()', () => {
           .to.equal(base64url.baseEncode(encodedCredentialResponse));
       });
 
+      it('should allow write with author rule', async () => {
+        // scenario: alice posts an image on the social mebia protocol, then she adds a caption
+        const protocol = 'https://tbd.website/decentralized-web-node/protocols/social-mebia';
+        const protocolDefinition: ProtocolDefinition = socialMebiaProtocolDefinition;
+
+        const alice = await TestDataGenerator.generatePersona();
```

Comment:
The test is not valid because Alice is writing to her own DWN so even without the `author` rule, the test will pass.

So you'll need have to involve Bob somehow in the scenario.

Reply:
Fixed